packages/shared/friends/sagas.ts

```diff
@@ -389,18 +389,28 @@ function* configureMatrixClient(action: SetMatrixClient) {
     handleIncomingFriendshipUpdateStatus(FriendshipAction.REJECTED, socialId)
   )
 
+  client.onChannelMembers((conversation, members) => {
+    if (conversation.name && !conversation.name.startsWith('Empty room')) {
+      updateChannelInfo(conversation, client)
+    }
+
+    const userStatuses = client.getUserStatuses(...members.map((member) => member.userId))
+    const membersPayload = members
+      .filter((member) => userStatuses.get(member.userId)?.presence === PresenceType.ONLINE)
+      .map((member) => ({ ...member, isOnline: true }))
```

Comment:
I'm wondering if there's a way we can reuse the logic from the `getOnlineMembers` or some intermediate function.